sdk/search/azure-search-documents/azure/search/documents/_service/_search_service_client.py

```diff
@@ -411,3 +412,145 @@ def create_or_update_synonym_map(self, name, synonyms, **kwargs):
         synonym_map = SynonymMap(name=name, synonyms=solr_format_synonyms)
         result = self._client.synonym_maps.create_or_update(name, synonym_map, **kwargs)
         return listize_synonyms(result.as_dict())
+
+    # Skillset Operations
+
+    @distributed_trace
+    def list_skillsets(self, **kwargs):
+        # type: (**Any) -> List[Skillset]
```

Comment:
Did we talk about making this return a `Pageable[T]`?

Reply:
I don't recall, @brjohnstmsft any opinion here? (is there any chance this API will ever be paged on the service side?)

Reply:
My guess would be no, given that the max number of skillsets is currently [200](https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/search/search-limits-quotas-capacity.md#indexer-limits), but @bleroy or @arv100kri may have more informed opinions about where things are headed in the future.

Reply:
I agree that sounds overkill for now.

Reply:
@brjohnstmsft in this case we would want to rename to `get_skillsets`, is that acceptable?